apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/TimedBufferingConsumer.cs

```diff
@@ -48,12 +48,10 @@ internal TimedBufferingConsumer(IConsumer<T> consumer, TimeSpan waitTime, IThrea
                 {
                     Flush();
                 }
-                catch (RpcException)
+                catch (Exception)
                 {
                     // TODO(talarico): This is a short term solution to ensure 
                     // we do not kill a process. See issue #842 to track the long term solution.
```

Comment:
Done.  Sorry I should have added more context here.  These are being done in multiple PRs for multiple releases.  I am merging this change into `diag-2.0.x` not master `diag-2.0.x`  this will be a release of 2.0 we are currently working on 3.0 now to fix a fairly bad issues that can occur during a stackdriver outage.